lib/screens/music_screen.dart

```diff
@@ -22,6 +23,29 @@ import '../services/finamp_settings_helper.dart';
 import '../services/finamp_user_helper.dart';
 import '../services/jellyfin_api_helper.dart';
 
+void postLaunchHook(WidgetRef ref) async {
+  final downloadsService = GetIt.instance<DownloadsService>();
+  final queueService = GetIt.instance<QueueService>();
+
+  // make sure playlist info is downloaded for users upgrading from older versions and new installations AFTER logging in and selecting their libraries/views
+  if (!FinampSettingsHelper.finampSettings.hasDownloadedPlaylistInfo) {
+    // check if metadata already downloaded to avoid possible duplicates
```

Comment:
Is this status check actually necessary?  We shouldn't ever run this block more than once due to the setting, and we shouldn't be able to produce duplicates of this anyway, The existing one should just be updated with the new profile and resynced.

Reply:
I wasn't sure how duplicated are handled, the check was just in case a user had already manually downloaded the metadata. If it's fine to re-add it, we can get rid of the check :)

Reply:
Yeah, everything should operate fine.  I don't think it's been possible to re-add something required before, but adding something that already has info uses basically the same update/overwrite path and that happens all the time without issue.

Reply:
I'll give it another try without the check tomorrow 👍🏻

Reply:
yep, seems to behave the same!